library/control/src/modules/WorkflowManager.rb

```diff
@@ -1604,12 +1608,18 @@ def package_repository(package_name)
       if pkgs.empty?
         log.warn("The installer extension package #{package_name} was not found")
         return nil
-      elsif pkgs.size > 1
+      end
+
```

Comment:
NP: What about adding the bug number here as reference

Reply:
I think the code it self-explaining and does not need any bug reference. If needed you can find it in the Git history.